src/sentry/models/grouptombstone.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+import logging
+
+from django.db import models
+
+from sentry.constants import LOG_LEVELS, MAX_CULPRIT_LENGTH
+from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model
+
+
+class GroupTombstone(Model):
+    __core__ = False
+
+    previous_group_id = BoundedPositiveIntegerField(unique=True)
+    project = FlexibleForeignKey('sentry.Project')
```

Comment:
i think django automatically creates indices for foreign keys

from [the docs](https://docs.djangoproject.com/en/1.7/ref/models/fields/#foreignkey):
> A database index is automatically created on the ForeignKey. You can disable this by setting db_index to False. You may want to avoid the overhead of an index if you are creating a foreign key for consistency rather than joins, or if you will be creating an alternative index like a partial or multiple column index.

and [source](https://github.com/django/django/blob/1.6.11/django/db/models/fields/related.py#L1166) since there aren't docs for our version anymore.

also, as a sanity check, looks like it from running migration locally:
![screenshot 2017-07-10 15 56 19](https://user-images.githubusercontent.com/5026776/28043912-22c10ddc-6589-11e7-8da6-669468a2b848.png)